### ISSUE TITLE
Supporting GitLab

### ISSUE THREAD

**Issue by manuelmerkel** [adr/adr-manager#105]

**Comment by ckuetbach** [comment]
Is this feature linked with this commit?

https://github.com/adr/adr-manager/commit/9db9fd30840dddbbe534d9d8c4cbd5c3837ca757

https://github.com/adr/adr-manager/compare/Issue-%23116

I would really like to see a gitlab connection, but it looks, like this feature is not merged or finished yet.

**Comment by xJREB** [comment]
Yes, unfortunately, this was not successfully implemented back then. However, a new wave of development will very likely start soon, and we can try to prioritize different Git backends like GitLab or GitHub Enterprise this time. Additionally, there is also the VS Code plugin, which is of course independent of any Git backend: https://github.com/adr/vscode-adr-manager

**Comment by ckuetbach** [comment]
That are good news. I'm using the vs Code plugin already, thats how I found this project.

But I would like to create a kind of plugin for https://backstage.io to encurage other to create ADR for their projects, too.

**Comment by Midnighter** [comment]
That sounds interesting @ckuetbach.
